packages/edit-site/src/components/global-styles/screen-typography-element.js

```diff
@@ -34,6 +45,7 @@ function ScreenTypographyElement( { name, element } ) {
 			<ScreenHeader
 				title={ elements[ element ].title }
 				description={ elements[ element ].description }
+				hint={ elements[ element ].hint }
```

Comment:
It may be worth using optional chaining here so it's clearer that `.hint` may not be defined.
```suggestion
				hint={ elements[ element ]?.hint }
```